debug_toolbar/middleware.py

```diff
@@ -78,6 +79,8 @@ def __init__(self, get_response):
             markcoroutinefunction(self)
 
     def __call__(self, request):
+        # Start tracking middleware execution time
+        toolbar_start_time = perf_counter()
```

Comment:
Sorry, I didn't go through this well enough earlier. What do you think about putting this start time in `TimerPanel.enable_instrumentation`? That way we put all of the timing logic in that specific panel and we avoid leaking it into the toolbar itself.

Reply:
Yes, added it to `aenable_instrumentation` as well